go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.6.1
+	go.uber.org/zap v1.10.0
```

Comment:
I don't think this is still needed.

Reply:
Yep, you are right. Just ran `go mod tidy` and it's gone.